.storybook/preview.js

```diff
@@ -1,14 +1,26 @@
 import { addDecorator, addParameters } from '@storybook/html';
 import { withA11y } from '@storybook/addon-a11y';
-import centered from '@storybook/addon-centered/html';
+import { withPaddings } from 'storybook-addon-paddings';
 import * as colors from '../src/design-tokens/colors.yml';
+import * as ms from '../src/design-tokens/modular-scale.yml';
 import 'focus-visible';
 import './preview.scss';
 
-// Add decorators to all stories
-addDecorator(centered);
+// Accessibility testing via aXe
 addDecorator(withA11y);
 
 // Theme selection from stories
 const themes = [{ name: 'Dark', class: 't-dark', color: colors.primaryBrand }];
 addParameters({ themes });
+
+// Padding values from modular scale
+const paddings = [];
```

Comment:
Nit, optional approach:

```js
const paddings = new Array(10).fill(0).map((k,i) => {
  const idx = i - 3;
  return { 
    name: `Step ${idx}`,
    value: `${Math.pow(ms.ratio, idx).toFixed(2)}em`,
    default: idx === 0
  };
});
```

Reply:
Not absolutely sure, but think the `+` for casting isn't needed before `Math.pow`

Reply:
@Josh68 I find that alternative approach a lot harder to read. It's also slightly more code, so I kept the version I have.

I added a comment explaining what `+` is doing here (it prevents numbers that don't need decimals like `1.00` from having them).

Reply:
You're right, it is harder to read. I've gotten used to preferring array helpers over loops, but in this case, the loop makes perfect sense.